x-pack/legacy/plugins/siem/common/constants.ts

```diff
@@ -56,6 +56,7 @@ export const DETECTION_ENGINE_PREPACKAGED_URL = `${DETECTION_ENGINE_RULES_URL}/p
 export const DETECTION_ENGINE_PRIVILEGES_URL = `${DETECTION_ENGINE_URL}/privileges`;
 export const DETECTION_ENGINE_INDEX_URL = `${DETECTION_ENGINE_URL}/index`;
 export const DETECTION_ENGINE_TAGS_URL = `${DETECTION_ENGINE_URL}/tags`;
+export const DETECTION_ENGINE_RULES_STATUS = `${DETECTION_ENGINE_URL}/rules/_find_statuses`;
```

Comment:
Can this be updated to utilize 
https://github.com/elastic/kibana/blob/535700a1b8ae9a8d792b6b45cc81c0912fb5e733/x-pack/legacy/plugins/siem/common/constants.ts#L54

so the new const would just be `${DETECTION_ENGINE_RULES_URL}/_find_statuses`?